src/frontend/js/app/routes.js

```diff
@@ -1,7 +1,7 @@
 import App from './App';
-import productRoutes from '../product/productRoutes';
-import orderRoutesFactory from '../order/orderRoutes';
-import userRoutes from '../user/userRoutes';
+import productRoutes from '../product/routes';
```

Comment:
Good question, can't remember. If we decide to move to components, I suggest we do so in a separate PR